openshift/templates/jobs/backups/README.md

```diff
@@ -0,0 +1,111 @@
+# How to Configure Mongo Database Backup Cronjob and Restore From it
+
+The OpenShift template that we are currently using to run our mongo database backups is ```backup-cron.yaml``` in this directory.
+This is based on the backup-container common component: https://developer.gov.bc.ca/Backup-Container 
+
+## How to Create a Mongo Database Backup Cronjob
+
+Note that the template assumes there is a service called ```landuseplanning-api-mongodb``` that is exposes the ```landuseplanning-api-mongodb``` deployment's pod locally. The template also relies on some OpenShift secrets and config maps that will need to be created.
+
+### Through the OpenShift UI or otherwise, you should to create:
+
+OpenShift secret named ```artifactory-creds``` (documentation for artifactory found here: https://developer.gov.bc.ca/Artifact-Repositories-(Artifactory) )
+
+
+OpenShift config map named ```backup-lup-conf``` using the ```backup-configmap.yaml``` file in this directory.
+
+Two PVCs (one for backups, one for verification) using the ```backup-pvc.yaml``` and ```verification-pvc.yaml```
+
+
+### To create the cronjob:
+
+Log in through the UI, and navigate to Workloads->CronJobs.
+
+Create a new CronJob using the ```backup-cron.yaml``` file in this directory, making any changes as required for your environment.
+
+The cron has been set to run daily at 0 9 * * * (9am UTC, 1am PST)
+
+## How to Restore a Mongo Database
+(these are old insstructions from a previous version of the backups, but should still apply)
+
+### Mount your backup PVC
+
+mount your backup PVC to a container so that you can access your backups. I use PVC migrator in this example (https://github.com/BCDevOps/StorageMigration/tree/master/openshift/templates) but you can use anything with a mount point. there should be a build of PVC migrator in the tools namespace so you should just have to deploy it in your working namespace from there.
```

Comment:
```suggestion
Mount your backup PVC to a container so that you can access your backups. I use PVC migrator in this example (https://github.com/BCDevOps/StorageMigration/tree/master/openshift/templates) but you can use anything with a mount point. there should be a build of PVC migrator in the tools namespace so you should just have to deploy it in your working namespace from there.
```